projects/angular/src/lib/maskito.directive.ts

```diff
@@ -6,29 +6,49 @@ import {
     NgZone,
     OnChanges,
     OnDestroy,
+    Optional,
+    Self,
 } from '@angular/core';
+import {DefaultValueAccessor} from '@angular/forms';
 import {
     Maskito,
     MASKITO_DEFAULT_ELEMENT_PREDICATE,
     MASKITO_DEFAULT_OPTIONS,
     MaskitoElementPredicate,
     MaskitoOptions,
+    maskitoTransform,
 } from '@maskito/core';
 
 @Directive({standalone: true, selector: '[maskito]'})
 export class MaskitoDirective implements OnDestroy, OnChanges {
     private maskedElement: Maskito | null = null;
 
     @Input()
-    maskito: MaskitoOptions | null = MASKITO_DEFAULT_OPTIONS;
+    maskito: MaskitoOptions | null = null;
 
     @Input()
     maskitoElement: MaskitoElementPredicate = MASKITO_DEFAULT_ELEMENT_PREDICATE;
 
     constructor(
         @Inject(NgZone) private readonly ngZone: NgZone,
         @Inject(ElementRef) private readonly elementRef: ElementRef<HTMLElement>,
-    ) {}
+        @Inject(DefaultValueAccessor)
+        @Self()
+        @Optional()
+        accessor: DefaultValueAccessor | null,
+    ) {
+        if (accessor !== null) {
```

Comment:
nit: we can save a byte or two by writing just `if (accessor)` :)

Reply:
```suggestion
        if (accessor) {
```